src/pages/Activity/ActivityCard.tsx

```diff
@@ -8,44 +8,42 @@ export function ActivityCard({ content }: ActivityCardProps) {
   return (
     <Box
       key={content.id}
-      border="1px solid"
-      borderColor="grey.200"
-      padding={3}
-      borderRadius={3}
+      sx={{
+        border: "1px solid",
+        borderColor: "grey.300",
+        borderRadius: 2,
+        p: 2,
+        backgroundColor: "white",
+        boxShadow: "0px 1px 4px rgba(0,0,0,0.1)",
+        display: "flex",
+        flexDirection: "column",
+        gap: 1.5,
+      }}
     >
-      <Box mb={1} display="flex" justifyContent="space-between">
-        <Typography fontWeight="bold" variant="subtitle1">
+      <Box display="flex" justifyContent="space-between" alignItems="center">
+        <Typography variant="subtitle1" fontWeight="bold" color="teal">
```

Comment:
No loose colors
Se a ideia era manter o verde tradicional, não é necessário especificar a cor
```suggestion
        <Typography variant="subtitle1" fontWeight="bold">
```